index.html

```diff
@@ -4,11 +4,12 @@
 
 <section class="greeting">
     <h1>
-        <img src="/assets/rust-analyzer.svg" alt="rust analyzer" style="width: calc(20% + 250px)">
+        <img id="logo" src="/assets/rust-analyzer.svg" alt="rust analyzer">
     </h1>
 
     <p>
-        Bringing <em>great</em> IDE experience to the Rust programming language.
+        Bringing <i style="color: #00a77e">great</i> IDE experience<br>
```

Comment:
I think I'd rather avoid the color. It certainly brings attention to the right thing, but I'd love the overall feeling to be boring and professional, rather than flamboyant.

Reply:
Okay, I changed it back.